src/stream_base.h

```diff
@@ -244,6 +244,7 @@ class StreamResource {
   // `*bufs` and `*count` accordingly. This is a no-op by default.
   // Return 0 for success and a libuv error code for failures.
   virtual int DoTryWrite(uv_buf_t** bufs, size_t* count);
+  virtual inline bool HasDoTryWrite() const { return false; }
```

Comment:
Can you please add a comment above this line explaining the semantics, given that `DoTryWrite` always exists?